src/App.js

```diff
@@ -11,6 +12,10 @@ function App() {
       <Nav></Nav>
       <Routes>
         <Route path="/reviews" element={<ReviewList></ReviewList>}></Route>
```

Comment:
Not necessary, but some people like using self closing tags on elements.

For example, instead of:
<ReviewList></ReviewList>
You can use:
<ReviewList />